2014-15/B02 Process Maternity (SMR02) extract.sps

```diff
@@ -292,10 +292,10 @@ save outfile = !file + 'maternity_for_source-20' + !FY + '.zsav'
     uri
     /zcompressed.
 
-get file = !file + 'maternity_for_source-20' + !FY + '.zsav'.
+get file = !Year_dir + 'maternity_for_source-20' + !FY + '.zsav'.
 
 * zip up the raw data.
-Host Command = ["gzip '" + !Extracts + "Maternity-episode-level-extract-20" + !FY + ".csv'"].
+Host Command = ["gzip '" + !Year_Extracts_dir + "Maternity-episode-level-extract-20" + !FY + ".csv'"].
```

Comment:
e.g. If we had no spaces in file paths this could be:
`["gzip " + !Year_Extracts_dir + "Maternity-episode-level-extract-20" + !FY + ".csv"].`

I think only a minor improvement so not worth prioritising just now!